iquip/apps/thread.py

```diff
@@ -65,4 +69,4 @@ def run(self):
                 ExperimentInfo(**experimentInfo)
             )
         else:
-            print("The selected item is a non-experiment file.")
+            logger.error("The selected item is a non-experiment file.")
```

Comment:
Although it is the original text, but I think "The selected item is not an experiment file." sounds better.

Reply:
How about modifying the level as `logging.info()`?
I think the case where the selected item is not an experiment file is not a critical problem.